File: Euler2.js
function euler2() {
	var sum = 0;
	var a = 0
	var b = 1
	for (var c = a + b; c < 4000000;){
		c = a + b;
		a = b;
		b = c;
		if (c % 2 == 0){
			console.log(c)
			sum += c
		}
	}
	return sum
}

console.log(euler2())

File: Euler 2/Euler2.java

public class Euler2 {
	public static void main(String[] args) {
		System.out.println(fibonacci());
	}
	
	private static int fibonacci() {
		int n1 = 1;
		int n2 = 2;
		int total = 0;
		while (n2 < 4000000) {
			if (n2 % 2 == 0) total += n2;
			int temp = n2;
			n2 += n1;
			n1 = temp;
		}
		return total;
	}
}
